src/components/form.jsx

```diff
@@ -46,7 +46,15 @@ export default function Form({ handleSubmit, form, onChange }) {
 				<button type="submit">Add Item</button>
 			</form>
 
-			{form.isSubmited && <div>item added succesfuly</div>}
+			{form.isSubmitted && (
+				<div>
+					{form.isLoading ? 'Loading...' : ''}
+					{
+						form.isSuccess ? 'Item added Successfully' : form.error
+					}
+					'Error adding item'
+				</div>
+			)}
```

Comment:
Liking the new user feedback, thanks for adding that in!

FYI on my end when I submit an item I am seeing:
'Loading...Error adding item'
'Item added SuccessfullyError adding item'

Should the 'Error adding item' string be the last piece of the form.isSuccess ternary operation, rather than out on it's own?

Reply:
I am also getting this bug now. Initially I wasn't because that text was commented out. I think React is not seeing it as pert of the ternary.